src/js/node/crypto.ts

```diff
@@ -10712,5 +10644,65 @@ crypto_exports.createVerify = createVerify;
   };
 }
 
+function randomBytes(size, callback) {
+  if (callback === undefined) {
+    return _randomBytes(size);
+  }
+
+  _randomBytes(size, callback).then(buf => {
+    callback(null, buf);
+  });
```

Comment:
should this have a `.catch` handler or is it guaranteed to succeed once it makes it to the Promise?

Reply:
the random promise is guaranteed to resolve. I'll leave a comment